### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by yevheniiIvanise**
Implemented: NavigationBar, Main section, Services section, some key components, css property values file

- [x] - RFR

### CODE REVIEW

File: config/paths.js [hunk 1]
```diff
@@ -26,9 +26,9 @@ const getPublicUrl = appPackageJson =>
   envPublicUrl || require(appPackageJson).homepage;
 
 // We use `PUBLIC_URL` environment variable or "homepage" field to infer
-// "public path" at which the app is served.
+// "public path" at which the Landing is served.
 // Webpack needs to know it to put the right <script> hrefs into HTML even in
-// single-page apps that may serve index.html for nested URLs like /todos/42.
+// single-Landing apps that may serve index.styl.html for nested URLs like /todos/42.
```
Comment (igor-shyrnin) [inline config/paths.js]: Remove changes in this file.

File: config/env.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ dotenvFiles.forEach(dotenvFile => {
 // It works similar to `NODE_PATH` in Node itself:
 // https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders
 // Note that unlike in Node, only *relative* paths from `NODE_PATH` are honored.
-// Otherwise, we risk importing Node.js core modules into an app instead of Webpack shims.
+// Otherwise, we risk importing Node.js core modules into an Landing instead of Webpack shims.
```
Comment (igor-shyrnin) [inline config/env.js]: Remove changes in this file.

File: config/webpack.config.prod.js [hunk 3]
```diff
@@ -229,7 +231,7 @@ module.exports = {
         console.log(message);
       },
       minify: true,
-      navigateFallback: publicUrl + '/index.html',
+      navigateFallback: publicUrl + '/index.styl.html',
```
Comment (igor-shyrnin) [inline config/webpack.config.prod.js]: Fot what you did that?

File: config/webpackDevServer.config.js [hunk 4]
```diff
@@ -41,15 +41,15 @@ module.exports = function(proxy, allowedHost) {
     // project directory is dangerous because we may expose sensitive files.
     // Instead, we establish a convention that only files in `public` directory
     // get served. Our build script will copy `public` into the `build` folder.
-    // In `index.html`, you can get URL of `public` folder with %PUBLIC_URL%:
+    // In `index.styl.html`, you can get URL of `public` folder with %PUBLIC_URL%:
```
Comment (igor-shyrnin) [inline config/webpackDevServer.config.js]: Remove changes in this file.

File: src/imports/components/NavigationBar/index.styl [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+.navigation-bar
+  display flex
+  justify-content flex-end
+  max-width 1432px
+  margin 32px auto 0px
+
+  .links
+    display flex
+    justify-content space-between
+    width 294px
```
Comment (igor-shyrnin) [inline src/imports/components/NavigationBar/index.styl]: You don't need to use it.

File: src/imports/components/SectionHeading/index.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import './index.styl';
+
+const SectionHeading = ({title}) => {
```
Comment (igor-shyrnin) [inline src/imports/components/SectionHeading/index.js]: Use:
```
const SectionHeading = ({title}) => (
  <div className="section-heading">
    {title}
  </div>
);
```

File: src/pages/Landing/index.styl [hunk 7]
```diff
@@ -0,0 +1,5 @@
+@import "~styles/config.styl"
+
+.landing
+  width 100%
```
Comment (igor-shyrnin) [inline src/pages/Landing/index.styl]: You don't need it.

File: src/pages/Landing/sections/Main/components/Description/index.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './index.styl';
+
+const Description = ({title, text}) => {
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Main/components/Description/index.js]: Try as above

File: src/pages/Landing/sections/Main/components/Description/index.styl [hunk 9]
```diff
@@ -0,0 +1,13 @@
+@import "~styles/config.styl"
+
+.description
+  display flex
+  width 470px
+  color: $very-dark-gray
+  line-height $normal-height
+  margin 226px 0 0 0
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Main/components/Description/index.styl]: Too big margin here. Use space-between better.

File: src/pages/Landing/sections/Main/components/Logo/index.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './index.styl';
+
+const Logo = ({description}) => {
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Main/components/Logo/index.js]: Try as above.

File: src/pages/Landing/sections/Main/components/Logo/index.styl [hunk 11]
```diff
@@ -0,0 +1,25 @@
+@import "~styles/config.styl"
+
+.logo
+  display flex
+  align-items center
+  align-self center
+  justify-content space-around
+  width 500px
+  height 230px
+  background-image: url("../../../../../../imports/images/logo-bg.png");
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Main/components/Logo/index.styl]: Use absolute path

File: src/pages/Landing/sections/Main/index.styl [hunk 12]
```diff
@@ -0,0 +1,8 @@
+@import "~styles/config.styl"
+
+.main
+  display flex
+  flex-direction column
+  justify-content space-between
+  max-width 1432px
+  margin 218px auto 0
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Main/index.styl]: Too big margin here. Use space-between better or another flex positional alignment

File: src/pages/Landing/sections/Services/components/Service/index.js [hunk 13]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import './index.styl';
+
+const Service = ({title, description}) => {
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Services/components/Service/index.js]: Try as above.

File: src/pages/Landing/sections/Services/index.styl [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+  flex-direction column
+  justify-content space-between
+  margin 80px 0 0 0
+
+  .service-list
+    display flex
+    justify-content space-around
+    align-items center
+    height calc(100vh - 92px)
+    background url("../../../../imports/images/services-bg.png") no-repeat center;
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Services/index.styl]: Use absolute path

File: src/imports/components/Footer/index.styl [hunk 15]
```diff
@@ -0,0 +1,17 @@
+@import "~styles/config.styl"
+
+.footer
+  display flex
+  justify-content space-between
+  max-width 1432px
+  margin 0 auto 26px
```
Comment (igor-shyrnin) [inline src/imports/components/Footer/index.styl]: Change to margin-bottom 26px

File: src/imports/components/NavigationBar/index.js [hunk 16]
```diff
@@ -2,17 +2,17 @@ import React from 'react';
...
-const NavigationBar = () => {
-  return (
-    <div className="navigation-bar">
-      <div className="links">
-        <a href="">Services</a>
-        <a href="">Process</a>
-        <a href="">Work</a>
-        <a href="">Contact</a>
-      </div>
+//TODO for changes is coming soon. Temporary implementation.
```
Comment (igor-shyrnin) [inline src/imports/components/NavigationBar/index.js]: Remove comment

File: src/imports/components/SectionHeading/index.styl [hunk 17]
```diff
@@ -11,3 +11,6 @@
   line-height $section-heading-height
   background: url("../../images/section-heading-bg.png") no-repeat center;
 
+.right-hand
+  margin 0 0 0 200px
```
Comment (igor-shyrnin) [inline src/imports/components/SectionHeading/index.styl]: Change to margin-left 200px

File: src/imports/components/TextArea/index.js [hunk 18]
```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+
+import './index.styl';
+
+class TextArea extends Component {
+  render() {
+    return (
+      <div className="form-text-area">
+        <textarea name="" id="" cols="30" rows="10" />
```
Comment (igor-shyrnin) [inline src/imports/components/TextArea/index.js]: Remove name and id

File: src/pages/Landing/sections/ContactUs/index.js [hunk 19]
```diff
@@ -0,0 +1,44 @@
...
+import TextArea from 'components/TextArea';
+
+import './index.styl';
+
+class ContactUs extends Component {
+  render() {
+    return (
+      <div className="contact-us" id="contact-us">
+        <SectionHeading title="Contact Us" />
+        <form action="" className="contact-us-form">
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/ContactUs/index.js]: why action is empty?

File: src/pages/Landing/sections/ContactUs/index.styl [hunk 20]
```diff
@@ -0,0 +1,21 @@
+@import "~styles/config.styl"
+
+.contact-us
+  display flex
+  flex-direction column
+  max-width 1432px
+  height calc(100vh - 76px)
+  margin 80px auto 0
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/ContactUs/index.styl]: Use margin top 80px

File: src/pages/Landing/sections/ContactUs/index.styl [hunk 21]
```diff
@@ -0,0 +1,21 @@
...
+
+  .contact-us-form
+    display flex
+    flex-direction column
+    align-self center
+    justify-content space-between
+    margin 80px 0 0 0
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/ContactUs/index.styl]: Same here

File: src/pages/Landing/sections/Main/components/Logo/index.styl [hunk 22]
```diff
@@ -2,12 +2,13 @@
...
   display flex
-  align-items center
+  flex 1
   align-self center
+  align-items center
   justify-content space-around
   width 500px
   height 230px
-  background-image: url("../../../../../../imports/images/logo-bg.png");
+  background: url("../../../../../../imports/images/logo-bg.png") no-repeat center;
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Main/components/Logo/index.styl]: Use absolute path

File: src/pages/Landing/sections/Main/index.js [hunk 23]
```diff
@@ -8,7 +8,7 @@ class Main extends Component {
   render() {
     return (
       <div className="main">
-        <Logo description="We Create Web & Mobile Applications That’ll Delight Your Users" />
+        <Logo motto="We Create Web & Mobile Applications That’ll Delight Your Users" />
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Main/index.js]: Use 'description' better.

Comment (igor-shyrnin) [reply src/pages/Landing/sections/Main/index.js]: or desc

File: src/pages/Landing/sections/Services/index.styl [hunk 24]
```diff
@@ -11,4 +11,5 @@
     justify-content space-around
     align-items center
     height calc(100vh - 92px)
-    background url("../../../../imports/images/services-bg.png") no-repeat center;
+    background url("../../../../imports/images/services-bg.png") no-repeat center
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Services/index.styl]: Use absolute path

File: src/pages/Landing/sections/TheProcess/components/Process/index.styl [hunk 25]
```diff
@@ -1,4 +1,34 @@
...
 .process
-  width 400px
+  display flex
+  flex-direction column
+  width fit-content
+
+  .process-circle
+    width 10px
+    height 10px
+    margin 0 0 134px 36px
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/TheProcess/components/Process/index.styl]: Try without margin.

File: src/pages/Landing/sections/TheProcess/index.styl [hunk 26]
```diff
@@ -3,4 +3,5 @@
 .the-process
   display flex
   flex-direction column
-  margin 80px 0 0 0
+  max-width 1200px
+  margin 80px auto 0
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/TheProcess/index.styl]: Use margin-top 80px

File: src/pages/Landing/sections/Work/components/WorkItem/index.js [hunk 27]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './index.styl';
+
+//TODO for changes is coming soon. Temporary implementation.
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Work/components/WorkItem/index.js]: Remove comment

File: src/pages/Landing/sections/Work/components/WorkItem/index.styl [hunk 28]
```diff
@@ -0,0 +1,13 @@
+@import "~styles/config.styl"
+
+.work-item
+  display flex
+  flex 1
+  justify-content center
+
+  span
+    font-size $work-item
+    line-height $work-item-height
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Work/components/WorkItem/index.styl]: Use &:last-child

File: src/pages/Landing/sections/Work/index.js [hunk 29]
```diff
@@ -0,0 +1,32 @@
...
+          <div className="work-list">
+            <WorkItem
+              title="CET"
+              color="#7ED321"
+            />
+            <WorkItem
+              title="BetterYet"
+              color="#FFDF4A"
+            />
+            <WorkItem
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Work/index.js]: Write it in one line

File: src/pages/Landing/sections/Work/index.styl [hunk 30]
```diff
@@ -0,0 +1,14 @@
...
+  display flex
+  flex-direction column
+  margin 234px auto 0
+
+  .work-list
+    display flex
+    justify-content space-around
+    align-items center
+    height calc(100vh - 92px)
+    background url("../../../../imports/images/services-bg.png") no-repeat center
```
Comment (igor-shyrnin) [inline src/pages/Landing/sections/Work/index.styl]: Use absolute path

File: src/imports/components/Footer/index.styl (hunk 15 above)
Comment (yevheniiIvanise) [reply src/imports/components/Footer/index.styl]: I totally agree with you about using margin-{side} css property type when we need to make one side margin. This case requires two indents at least.

File: src/pages/Landing/sections/TheProcess/index.styl (hunk 26 above)
Comment (yevheniiIvanise) [reply src/pages/Landing/sections/TheProcess/index.styl]: The same reason i mentioned above.

**Pull request merged**